src/usr/local/www/interfaces_assign.php

```diff
@@ -550,17 +621,23 @@ function interface_assign_description($portinfo, $portname) {
 			</th>
 			<td>
 				<select name="if_add" id="if_add" class="form-control">
-<?php foreach ($unused_portlist as $portname => $portinfo):?>
-					<option value="<?=$portname?>" <?=($portname == $iface['if']) ? ' selected': ''?>>
-						<?=interface_assign_description($portinfo, $portname)?>
-					</option>
-<?php endforeach;?>
+<?php //$timea2 = microtime(true);
+/* As with the gettext() calls, I've removed the interface_assign_description() calls and
+replaced them with my own interface_assign_description_fast() function that's called once
+outside of the loop. Also like the gettext() edits, this change keeps paying for itself.
+ Also, removing the indents and newlines saves potentially ~9KB of HTML with 500 unassigned VLANs */
```

Comment:
Comment not helpful here. "HTML indenting not used to save tmie/space" would suffice.